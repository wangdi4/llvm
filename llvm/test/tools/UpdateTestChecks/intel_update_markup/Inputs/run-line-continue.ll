; RUN: llc < %s \
; RUN:  | FileCheck %s
