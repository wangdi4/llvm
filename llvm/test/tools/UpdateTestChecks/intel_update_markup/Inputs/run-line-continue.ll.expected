; INTEL_CUSTOMIZATION
; RUN: llc < %s \
; RUN:  | FileCheck %s
; end INTEL_CUSTOMIZATION
